src/Microsoft.Identity.Web.MicrosoftGraph/Constants.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// General constants.
+    /// </summary>
+    internal static class Constants
+    {
+        public const string Authorization = "Authorization";
+        public const string Bearer = "Bearer";
+
+        // Microsoft Graph
+        public const string UserReadScope = "user.read";
+        public const string GraphBaseUrlV1 = "https://graph.microsoft.com/v1.0";
+    }
+}
```

Comment:
hmmmm...not a fan of repeating code. especially for maintainability and fixes. i can look into how to resolve this.

Reply:
Constants class in Identity.Web are internal. Perhaps set InterntalsVisibleTo to Identity.Web.Graph?

Reply:
Thanks you for the PR comments. Addressed.